README.md

```diff
@@ -72,6 +72,25 @@ Enums.Framerate framerate = timecodeObj.Framerate;
 string timecodeRegex = Timecode.RegexPattern;
 ```
 
+### Operator Overloading
+
+```csharp
+
+var tc1 = new Timecode(1, 2, 20, 100, DotnetTimecode.Enums.Framerate.fps23_976);
+var tc2 = new Timecode(0, 22, 31, 100, DotnetTimecode.Enums.Framerate.fps23_976);
```

Comment:
As examples should be easy to understand I suggest we keep the timecode values simple.

Please change the values of tc1 and tc2 to 10:00:00:00 and 01:00:00:00, as it will make it easier for the consumer of the README.md file to understand what is going on. 

The current timecode constructors are valid, but a bit complex for the scope. For example, the timecode with the constructor Timecode(1, 2, 20, 100, Enums.Framerate.fps23_976) would result in a string value of somewhere around "01:02:24:04", which is rather confusing.

Reply:
Thanks, @RasmusBroborg for going through the request and letting me know the changes required.
As suggested I have removed the extra space, changed the tc1 and tc2 values to 10:00:00:00 and 01:00:00:00, and also changed the output values accordingly in the new commit. Please check and let me know if there is anything else I need to do.